packages/server/app/Controllers/Http/MyProfileController.test.ts

```diff
@@ -63,47 +86,37 @@ test.group('Users Profile information tests', (group) => {
             followersCounter: 0,
             followingCounter: 0,
             playlistsCounter: 0,
-            userID,
-            userNickname,
+            userID: user.uuid,
+            userNickname: user.nickname,
         };
         assert.deepEqual(parsedBody, expectedBody);
     });
 
-    test('It should send back user not found', async () => {
-        const userID = datatype.uuid();
+    test('Returns a 401 error when the current user is unauthenticated', async () => {
```

Comment:
💭 thought: interesting on how adding the authentication changes our tests definition